INPUT_SCHEMA.json

```diff
@@ -102,16 +102,25 @@
         "saveSubsToKVS": {
             "title" : "Save downloaded subtitles to Key Value Store",
             "type": "boolean",
-            "description" : "If set to true, we will save downloaded (and converted to .srt format) subtitles to Key Value Store in format { subtitles: srt_text }. 'downloadSubtitles' must be set to true for this take an effect"
+            "description" : "If set to true, we will save downloaded (and converted to .srt format) subtitles to Key Value Store in format { subtitles: srt_text, language: languageCode, type:autoOrUserGenerated }. 'downloadSubtitles' must be set to true for this take an effect"
         },
         "subtitlesLanguage": {
-            "title" : "Download subtitles in selected language",
+            "title" : "Download subtitles in selected language.",
             "type": "string",
-            "description" : "Language to download subtitles in. 'downloadSubtitles' must be set to true for this take an effect",
-            "editor": "select",
-            "default" : "en",
-            "enum" : ["en", "de", "es", "fr", "it", "ja", "ko", "nl", "pt", "ru"],
-            "enumTitles" : ["English", "German", "Spanish", "French", "Italian", "Japanese", "Korean", "Holland", "Portuguese", "Russian"]
+            "description" : "Language to download subtitles in, using standardized language codes (eg. 'en', 'de', 'ru' and so on). 'downloadSubtitles' must be set to true for this take an effect. If set to `null` or empty string, we download all available subtitles.",
+            "default" : null,
+            "editor": "textfield"
+        },
+        "preferAutoGeneratedSubtitles": {
+            "title": "Prefer automatically generated subtitles before the user generated ones.",
+            "description": "If set to `true` and language is provided, we download automatically generated subtitles rather then user ones. If no language is provided, this settings takes no effect",
+            "type": "boolean"
+        },
+        "scrapeCommentCount": {
```

Comment:
I would simply call it `maxComments`. 
1. We call it like that elsewhere
2. `Scrape` is reduntant since it is a scraper
3. `commentCount` sounds like we return only the count